# List of current picks
PickState[] picks